src/panoramic/cli/field_mapper.py

```diff
@@ -1,9 +1,10 @@
-from typing import Optional, Tuple
+from typing import Dict, Optional, Tuple
 
 from panoramic.cli.field.client import Field
 from panoramic.cli.pano_model import PanoField
 
 NAMESPACE_DELIMITER = '|'
+_METRIC_DATA_TYPES = ['FLOAT', 'DOUBLE', 'DECIMAL']
```

Comment:
This isn't used is it?

Reply:
Good catch 👍

Reply:
Removed in 02c4313da153d6d1f11f7ce24a173f40f325230d.